testdata/sqllogictests_iceberg/local.slt

```diff
@@ -94,3 +94,9 @@ REG AIR    314
 SHIP       316
 TRUCK      264
 
+# We should be able to query a table with version number longer than i32 limit.
+query T
+select count(*) = 1000
+  from iceberg_scan('../../testdata/iceberg/tables/lineitem_simple_longversion');
+----
+t
```

Comment:
Something I've found useful when adding tests in response to issues is to link to the issue directly. Prevents us from accidentally deleting it in the future thinking some other test covers it or something.

Reply:
Makes sense! Will link it :)

Reply:
Updated!